.github/workflows/tests.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           python-version: '3.13'
 
       - id: setup-mongodb
-        uses: mongodb-labs/drivers-evergreen-tools@master
+        uses: mongodb-labs/drivers-evergreen-tools@6a0686e0194d30b0a259404839c3e8bf9be12a5b
```

Comment:
Does this actually work? My understanding from @alcaeus' comments in the meeting this morning was that we couldn't use a commit hash here and would need to resort to a submodule.

Reply:
The only issue it that we cannot use dependabot to track new versions.
> For each action in the file, Dependabot checks the action’s reference (typically a version number or commit identifier associated with the action)
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#about-dependabot-version-updates-for-actions

I tried [previously](https://github.com/mongodb/mongo-php-library/network/updates/944321909). 
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/c92ca20d-d357-432a-9cf5-25235b4eca45" />